.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -553,6 +578,13 @@ if [[ "$is_active_tmp" == "0" ]]; then
 fi
 # END TO BE DELETED
 
+if [ "${WORKER}" == 0 ]; then
+    # Stop all foreign worker daemons
+    echo -e "\nStopping worker machine daemons"
+    sudo -H -u ${HWCRON_USER} ${SUBMITTY_INSTALL_DIR}/sbin/shipper_utils/systemctl_wrapper.py stop --target perform_on_all_workers
+    echo -e "\n"
+fi
```

Comment:
Why is this necessary? Given that we stop the shipper/worker on `INSTALL_SUBMITTY.sh`, why do we need to stop the workers here explicitly instead of just letting the `INSTALL_SUBMITTY.sh` process handle that (and restarting the workers if they're running.